pkg/controller/jenkins/reconciler.go

```diff
@@ -153,14 +150,21 @@ func (r *JenkinsReconciler) Reconcile(request reconcile.Request) (reconcile.Resu
 	// TODO implement error checking for existence of the resource
 	// *or if there is an actual error* deal with it (<>_<>-)
 	// Also -- https://github.com/redhat-developer/openshift-jenkins-operator/pull/17#pullrequestreview-289463590 --
-
+	errors := new(utils.Errors)
```

Comment:
Similarly we would have to initialize this a little early on with r.errors instead. And can you replace new() with &utils.Errors{} because it feels like that kind of allocation is used everywhere else. Else we can refactor the code to use new() as it is more readable :thinking:

Reply:
Fixed